aip/general/0122.md

```diff
@@ -282,6 +282,11 @@ When a field represents another resource, the field **should** be of type
 `string` and accept the resource name of the other resource. The field name
 **should** be equivalent to the corresponding message's name in snake case.
 
+The field **should not** be of type `message` using the `message` that 
+implements the resource, _unless_ the API is internal-only, has tight lifecycle
+relationships, and has a permission model that enables inherited access to
+embedded resources.
+
```

Comment:
Will adding examples help or clutter the AIP?

Also farther up under the parent section, there's a phrase `If there is more than one possible parent type`.  I thought there can only one parent?

Reply:
I think this might clutter the AIP, and I definitely plan to have proto examples in the linter rule documentation folks are linked to from the findings. I don't tend to include proto examples in AIPs of what *not* to do since folks often skim an AIP and latch onto examples :)

> `If there is more than one possible parent type`. I thought there can only one parent?

The key part of this is `parent **type**` - a resource instance will always have exactly one parent resource instance, but there could be multiple resource types suitable for parenting the resource.